cfn-resources/go.mod

```diff
@@ -10,7 +10,28 @@ require (
 	github.com/rs/xid v1.2.1
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cast v1.3.1
-	go.mongodb.org/atlas v0.5.1-0.20201123170532-371c0d8194fe
+	go.mongodb.org/atlas v0.17.0
```

Comment:
[q] Any reason why you are not using [v1.18](https://github.com/mongodb/go-client-mongodb-atlas/releases)?

Reply:
Upgraded to 0.18.0 in the latest commit.